src/Makefile

```diff
@@ -65,8 +65,10 @@ OSSEC_LDFLAGS=${LDFLAGS} -lm
 ifneq (${TARGET},winagent)
 ifeq (${uname_S},Linux)
 		DEFINES+=-DINOTIFY_ENABLED
-#		DEFINES+=-DUSE_MAGIC
 		OSSEC_LDFLAGS+=-lpthread
+		DEFINES+=-DHAVE_SYSTEMD
```

Comment:
Should this be conditional somehow for older linux systems?

Reply:
Yes, yes it should. My makefile-fu is weak, barely grasshopper status, but IIUC this would happen when configure/autogen determine if the systemd headers are available to compile against. Wanna toss me a PR? :)

Reply:
There's no configure/autogen. I'll try to get you a diff soon.

Reply:
I started a PR, but it looks like your tree is older. So the PR would be huge with a bunch of stuff unrelated to this. It's the last few commits [here](https://github.com/ddpbsd/ossec-hids/tree/reader_journald). If there's a better way to get you a diff, let me know.

Reply:
Updated my GH master, so now my branch is equivalent to this PR and yours should apply cleanly. Thank you

Reply:
Solved - cherry-picked and squashed your commits locally, rebased on master, force-pushed up. Thank you sir.